assets/css/structures/_header.scss

```diff
@@ -1,4 +1,4 @@
-.title__thicc {
+.title--thicc {
```

Comment:
These styles are doubled. They exist in the typography partial as well as here. I think this is the more up-to-date version. Can you move these into typography and delete them here.

Maybe we should also make the naming of headlines more consistent. Currently, we have `title--thicc`, `main-headline` and `subtitle`. I would propose `thicc-headline`, `main-headline`, and `sub-headline`.